1-resilient-messaging/src/Event/OrderWasPlaced.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Event;
+
+final class OrderWasPlaced
```

Comment:
Aby było PRO, możesz dodać `final readonly class`. Wtedy ten event będzie immutable :)

Reply:
Jako, że jest to event związany z naszą domeną, wrzuciłbym go do Domain (namespace/package). 
To już takie niuanse, które możemy podpowiedzieć ludzią, jakby pytali się jak można to rozwiązanie jeszcze ulepszyć :)